tasks.py

```diff
@@ -147,6 +149,34 @@ def release_notes(ctx, version=None, username=None, password=None, write=False):
     generator.generate(version, username, password, file)
 
 
+@task
+def build_libdoc(ctx):
+    """Update libdoc html template and language support.
+
+    Regenerates `libdoc.html`, the static template used by libdoc.
+
+    Update the language support by reading the translations file from the libdoc
+    web project and updates the languages that are used in the libdoc command line
+    tool for help and language validation.
+
+    This task needs to be run if there are any changes to libdoc.
+    """
+    subprocess.run(['npm', 'run', 'build', '--prefix', 'src/web/'])
+
+    src_path = Path(__file__).parent / "src" / "web" / "libdoc" / "i18n" / "translations.json"
+    data = json.loads(open(src_path).read())
+    keys = sorted([key.upper() for key in data.keys()])
```

Comment:
- No need to use `Path(__file__).parent` here. The `assert` at the top of the module makes sure this script can run only at the project root, so relative paths are fine.
- When using `Path` objects, `path / 'x/y/z'` is more convenient than `path / 'x' / 'y' / 'z'`.
- `open(path).read()` -> `path.read_text(encoding='utf-8')`
- `for key in data.keys()` -> `for key in data` in general.
- In this case `languages = sorted(data, key=str.upper)` is even better than the above.

Reply:
fixed all but the last, since I want the keys in the final list to be uppercased (they are not in the original dictionary)